azure-pipelines/publish-roslyn-copilot.yml

```diff
@@ -0,0 +1,58 @@
+trigger: none
+pr: none
+
+resources:
```

Comment:
Kind of a meta question here.  Should this pipeline even be in this repo?  Would conversations be a better place?

For example if you want to change the folder name where it finds the conversations build artifacts to upload.  Now you have to make the change across two repos.  If this was in conversations you just updated both at the same time.